docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -101,22 +112,22 @@ The workflow can take several minutes to run. While it is running, you can inves
      idaho_scheme:
        custom_sample:
          query: --query "(custom_data == 'yes')"
-         max_sequences: 5000
+         max_sequences: 50
```

Comment:
We should probably update the screenshot to reflect these changes. I will push a new commit.

Reply:
Good call! If you don't mind pushing an updated screenshot, that would be great.

Reply:
Done.